base.py

```diff
@@ -183,4 +187,34 @@ def __str__(self):
         tbp += 'Budget: ${:0,.2f}'.format(self.budget)+'\n'
         tbp += 'Profit: ${:0,.2f}'.format(self.getProfit())+'\n'
         tbp += '------'
-        return tbp
\ No newline at end of file
+        return tbp
+
+    def toJSON(self):
```

Comment:
Each feature should have its own PR. Make another PR for this, so this PR contains only login/registration

Reply:
I can't skipped the second commit, I've tried to create another branch for the PR and target into this branch but the files changes is a little bit weird.